cves/kernel/CVE-2020-27170.yml

```
@@ -149,10 +148,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Initially, when this was discovered, an embargo was put in place on who could access information concerning this vulnerability. 
+    But this vulnerability was discovered by an developer Piotr Krysiuk. However, the area where this vulnerability was found has been a problem area for quite a while dealing with spectre type vulnerabilities.
```

```suggestion
    This vulnerability was discovered by developer Piotr Krysiuk. However, the area where this vulnerability was found has been a problem area for quite a while and lead to many spectre type vulnerabilities.
```
Not sure if that fix at the end is what you meant, but if it isn't it might be a good idea to reword it a little.